internal/k8s/client.go

```diff
@@ -576,6 +586,13 @@ func (c *Client) GetPodsTable(ctx context.Context) (*metav1.Table, error) {
 	return unstructuredToTable(i[0].Object)
 }
 
+func (c *Client) ListUnstructured(ctx context.Context, gvr schema.GroupVersionResource, namespace *string, o metav1.ListOptions) (*unstructured.UnstructuredList, error) {
```

Comment:
Good job on figuring this one out! These things are a bit of a maze to navigate. I found that [`sigs.k8s.io/controller-runtime/pkg/client`](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client) is kind of best way to go about types, it accepts typed resources as well as `Unstructured`.